migrations/helpers/sample_timestamps_helper.py

```diff
@@ -0,0 +1,140 @@
+import datetime
+import re
+import sys
+import traceback
+
+CREATED_DATE_FIELD_NAME = 'First Imported Date' # TODO: check with E & A
+BATCH_SIZE = 250000
+COLLECTION_NAME_FORMAT_1 = r'^samples_20(\d){4}_(\d){4}$' # e.g. samples_200519_1510
+COLLECTION_NAME_FORMAT_2 = r'^samples_(\d){4}2020_(\d){4}$' # e.g. samples_21052020_1510
+
+
+# For updates, '$set' syntax is important, otherwise it overwrites the whole document!
+# TODO: check why _created and _updated are 1970, and whether I can insert into _created here, to back-fill data
+def add_timestamps_to_samples(db):
+  print(f'Time start: {datetime.datetime.now()}')
+
+  try:
+    print('\n-- Update samples collection with new concatenated id column --')
+    update_result_1 = db.samples.update_many(
+      { },
+      [
+        { '$set': { 'concat_id': { '$concat': [ "$Root Sample ID", " - ", "$RNA ID", " - ", "$Result", " - ", "$Lab ID" ] } } }
+      ]
+    )
```

Comment:
Probably a good idea, I will look into that. Wasn't sure if we wanted to keep it permanently, but I think it could save some headaches in the future? If we're keeping it, you guys will have to set it in your code too?